packages/app/cypress/e2e/runner/reporter.errors.cy.ts

```diff
@@ -0,0 +1,90 @@
+import { verify } from './support/verify-helpers'
```

Comment:
These tests were carried forward from the runner package and modified to be more e2e.